include/sys/dsl_scan.h

```diff
@@ -55,6 +56,7 @@ typedef struct dsl_scan_phys {
 	uint64_t scn_cur_max_txg;
 	uint64_t scn_start_time;
 	uint64_t scn_end_time;
+	uint64_t scn_scrub_pause_time; /* when scrub was paused */
```

Comment:
regardless of the other errata, you definitely can't change the offset of the following fields without adding a feature flag and managing the transition to the new on-disk format.